Makefile

```diff
@@ -12,7 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+.PHONY: push-multiarch-% push-multiarch
+
 CMDS=csi-node-driver-registrar
 all: build
 
 include release-tools/build.make
+
+# This target builds multiarch images using Moby BuildKit builder toolkit.
+# Docker Buildx is included in Docker 19.03 and needs DOCKER_CLI_EXPERIMENTAL enabled to run corresponding commands.
+# Currently amd, s390x and Windows manifest is pushed for canary, release branch and released tags.
+# Images generated from Prow build are pushed to staging area on gcr
+push-multiarch-%:
+	make BUILD_PLATFORMS="windows amd64 .exe"
+	set -ex; \
+	gcloud auth configure-docker; \
```

Comment:
This is going to require gcloud? Can we put this in the cloudbuild.yml instead?

Reply:
Earlier gcloud command was a part of cloudbuild only but later we have moved it to make target in order to keep cloudbuild simple and minimal.

Reply:
I think it would be preferable to keep gcloud command outside of this otherwise anybody that wants to build a multiarch image is going to require gcloud to be installed.

If you still want put it in the makefile to reduce the commands in cloudbuild, then I would maybe wrap "push-multiarch" with a "cloudbuild-push-multiarch"

Reply:
I agree with @msau42 that this belongs into cloudbuild.yml. `push-multiarch-%` should be independent of the registry. `cloudbuild.yml` is where we prepare for pushing to gcr.